tests/test_nodes.py

```diff
@@ -1947,3 +1947,22 @@ def test_match_class():
             case1.pattern.cls,
             *case1.pattern.kwd_patterns,
         ]
+
+    @staticmethod
+    def test_return_from_match():
+        code = textwrap.dedent(
+            """
+        def return_from_match(x):
+            match x:
+                case 10:
+                    return 10
+                case _:
+                    return -1
+
+        return_from_match()  #@
```

Comment:
Shouldn't 'x' be passed here ?
```suggestion
        return_from_match(y)  #@
```